sig-devops/README.md

```diff
@@ -9,6 +9,8 @@ The DevOps SIG is to design and implement the DevOps functionalities including p
 - Rick ([@linuxsuren](https://github.com/linuxSuRen/)), Member
 - Harrison Liu ([@harrisonliu5](https://github.com/harrisonliu5)), Member
 - John Niang ([@johnniang](https://github.com/JohnNiang)), Member
+- Abdelouahab Mbarki ([@AbdelouahabMbarki](https://github.com/AbdelouahabMbarki)), Member
+- mango ([@mangoGoForward](https://github.com/mangoGoForward)), Member
```

Comment:
@mangoGoForward Do you have an English name or other names that you want to use?

Reply:
Thanks,  `mango` is good.